src/store/rootReducer.ts

```diff
@@ -1,8 +1,10 @@
 import { combineReducers } from 'redux';
 import mainReducer from '../modules/main/store/mainSlice';
+import directionReducer from '../modules/direction/store/directionSlice';
 
 const rootReducer = combineReducers({
   main: mainReducer,
+  direction: directionReducer,
```

Comment:
i wrote it in Daria's PR:
![image](https://user-images.githubusercontent.com/22084409/100849494-55d43080-348b-11eb-9059-14171288c898.png)

can you pls sync with her?
so that we'd align the store in your changes